src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java

```diff
@@ -72,25 +76,27 @@ public void insertUpdate(final DocumentEvent de) {
 
             /**
              * Deletes the number of rows in Code Area Text.
+             *
              * @param de event when deleting text in Code Area Text.
              */
             @Override
             public void removeUpdate(final DocumentEvent de) {
                 lineCode.setText(getText());
             }
         });
-
         setLayout(new BorderLayout());
```

Comment:
Are they in method or constructor?

Reply:
It will be reviewed when CodetextArea is refactored.

Reply:
It´s already solved.